var/spack/repos/builtin/packages/iso-codes/package.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack import *
+
+
+class IsoCodes(AutotoolsPackage):
+    """ISO-codes provides lists of various ISO standards"""
+
+    homepage = "https://salsa.debian.org/iso-codes-team/iso-codes"
+    url      = "https://deb.debian.org/debian/pool/main/i/iso-codes/iso-codes_4.3.orig.tar.xz"
+
+    version('4.3', sha256='643eb83b2d714e8650ed7112706968d057bf5b101ba71c8ef219e20f1737b141')
+
+    depends_on('m4', type='build')
+    depends_on('autoconf', type='build')
+    depends_on('automake', type='build')
+    depends_on('libtool', type='build')
+    depends_on('gettext')
+    depends_on('python@3:')
```

Comment:
build, link, run?

Reply:
@adamjstewart I'm not sure how to answer that question -- I'd guess `build` only, based on `https://github.com/Homebrew/homebrew-core/blob/master/Formula/iso-codes.rb`, but it probably wouldn't hurt to check the pkgconfig file.

Reply:
@adamjstewart `gettext` and `python@3:` are `'build'` dependencies only. Updated in https://github.com/spack/spack/pull/12389/commits/d8c9fc2036f463d8869fc555b7bddcdb28daadda